apps/src/codebridge/FileBrowser/prompts/openNewFolderPrompt.tsx

```diff
@@ -31,19 +34,21 @@ export const validateNewFolderName = ({
   folderName,
   parentId,
   projectFolders,
-}: ValidateInputArgs) => {
+}: ValidateNewFolderNameArgs) => {
   if (!folderName.length) {
     return;
   }
   if (!isValidFolderName(folderName)) {
     return codebridgeI18n.invalidNameError();
   }
 
-  const existingFolder = Object.values(projectFolders).some(
-    f => f.name === folderName && f.parentId === parentId
-  );
-  if (existingFolder) {
-    return codebridgeI18n.folderExistsError();
+  const duplicate = checkForDuplicateFoldername({
+    folderName,
+    parentId,
+    projectFolders,
+  });
+  if (duplicate) {
+    return duplicate;
   }
 };
```

Comment:
This commit is a temporary simplification to consolidate logic into the `checkForDuplicateFoldername` function, which is deprecated in a later commit.